core-blocks/latest-comments/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import { DEFAULT_CONTROLS } from 'editor/components/block-alignment-toolbar';
```

Comment:
Per my other comment, we should get rid of this, but worth noting that this would've been a disapproved import (importing private entities across packages).

Reply:
I think I'm lost–how is it a private entity?

Reply:
Because it's not a top-level export of `editor`, i.e. you wouldn't be able to get it via

```js
import { DEFAULT_CONTROLS } from '@wordpress/editor';
```

and we avoid importing across blocks in any other fashion (i.e. a module in `core-blocks` importing from `editor`). There were some exceptions to this due to circular dependencies, but I think they're all but solved.

Reply:
Aha, gotcha.

Has there been discussion about a custom rule to catch that for us or has it been difficult to implement? Seems quite easy to miss in a review.

Reply:
There are [rules](https://github.com/WordPress/gutenberg/blob/73c24d83a3360be6852d33039327f660458553fc/.eslintrc.js#L41-L112) against `import { Bla } from 'editor';` (and any other package), but none for deep access. I'm assuming we don't yet forbit deep access because of those unresolved circular dependencies we had-and-may-still-have. Someone may correct me on this. :)

Reply:
We should update the rules and drop the `$`, or at least make it `(\\/|$)`

Reply:
Uncanny! Do you have a matcher for _lint_?

Reply:
That won't actually work (eslint throws an error), but I have a PR incoming.

Reply:
>That won't actually work (eslint throws an error), but I have a PR incoming.

Yeah, might be the same thing I've encountered before, where it was solved by including the unicode equivalent character of the slash.

Reply:
>Uncanny! Do you have a matcher for lint?

I actually have no idea how this notification came through to my inbox 😅

Reply:
Whoa, is that what that is? If it works I'll add a comment for it this time 😅

Reply:
zomg that worked, thanks. I'll add it and add a comment this time 😉